client/src/TagCopy.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './TagCopy.css';
+
+export default function Tag(props) {
```

Comment:
why is this file's name `TagCopy.js`?

Reply:
+1; we can just add to Tag.js and add the delete button there